client/state/data-layer/wpcom/sites/index.js

```diff
@@ -20,4 +123,5 @@ export default mergeHandlers(
 	plugins,
 	posts,
 	simplePayments,
+	siteHandlers
```

Comment:
Could we consider to move `siteHandlers` and everything that makes it work into a separate module? I think we should keep this file as an "entry point" as it was: just importing other submodules.